llumnix/arg_utils.py

```diff
@@ -200,9 +207,15 @@ def check_args(cls, args: 'EngineManagerArgs', parser: argparse.ArgumentParser):
             ("When using gloo as migration backend, "
              "do not set --disable-init-instance-by-manager and --disable-fixed-node-init-instance.")
 
+        # TODO[xinyi, kuilong]: grpc and kvtransfer can be used for vllm.
+        assert args.migration_backend not in ['kvtransfer'] or (args.migration_backend == 'kvtransfer' \
```

Comment:
We will provide some explanations in `cli_parser` and `Arguments.md`. Additionally, there is an assertion to establish a migration backend for specific engines to prevent unsupported parameters.